### PR TITLE
add option to pass custom Python interpreter

### PR DESCRIPTION
**Pull request by dirk-thomas**
Connect to ros2/ci#72

### CODE REVIEW

File: ament_tools/verbs/build_pkg/cli.py [hunk 1]
```diff
@@ -185,6 +185,13 @@ def add_arguments(parser):
         default=False,
         help='Use symlinks instead of copying files wherever possible',
     )
+    parser.add_argument(
+        '--python-interpreter',
+        default=sys.executable,
+        help='Use a different Python interpreter for invoking subprocesses'
+             " (default '%s')" % sys.executable,
```
Comment (wjwwood) [inline ament_tools/verbs/build_pkg/cli.py]: nitpick: using different quote types on strings that are implicitly concatenated looks weird, I'd recommend changing the first lint to use `"` too.

Comment (dirk-thomas) [inline ament_tools/verbs/build_pkg/cli.py]: Sure. Done.

**Comment by wjwwood** [comment]
Looks like CI is failing due to an unused import.

**Comment by dirk-thomas** [comment]
Fixed CI.

**Pull request merged**